dice.js

```diff
@@ -0,0 +1,8 @@
+const rollDice = function (rolling) {
+    for (let i = 0; i < rolling; i++) {
+        const num1 = Math.floor(Math.random() * 6) + 1;
+        const num2 = Math.floor(Math.random() * 6) + 1;
+        const sum = num1 + num2;
+        console.log("Dice rolled are " + (i+1) +" ,"+  num1 + ", and " + num2 + ". Sum is " + sum);
+    }
+}
```

Comment:
The bonus was asking to get the numbers of dice rolled as an arguments,
- so for example if you called the function like this `rollDice(2)` you should get ` Dice rolled are 1 and 3. Sum is 4` as result.
- and if you called it like this `rollDice(4)` you should get `Dice rolled are 1, 4, 2 and 3. Sum is 10`